Dockerfile

```diff
@@ -335,7 +335,7 @@ COPY --from=concourse-resource-builder /code/dist .
 
 RUN set -x \
     && . $PROJECT_ROOT/venv/bin/activate \
-    && pip3 install corgi-concourse-resource-*.tar.gz \
+    && pip3 install corgi_concourse_resource-*.tar.gz \
```

Comment:
I am not sure if the way poetry 1.4.0 names packages is intended. It goes against [package naming standards](https://peps.python.org/pep-0008/#package-and-module-names) and might actually be a bug. It could also be something we can fix in the pyproject.toml file, I am not sure. In the meantime, I wonder if it might be better to do something like:
```Dockerfile
&& pip3 install corgi*.tar.gz
```
There should only be one .tar.gz file in that directory.

Reply:
I think `pip3 install corgi*.tar.gz` is fine if there is anyway only one tar.gz file supposed to be in the directory. I've also had a "successful fail" with `pip3 install corgi*.tar.gz` on an empty directory (just to be sure it fails if the tar.gz is missing)

Reply:
I'll do `&& pip3 install corgi*concourse*resource*.tar.gz`